rules/falco_rules.yaml

```diff
@@ -2497,11 +2497,11 @@
   desc: Detect attempts to use K8s NodePorts from a container
   condition: (inbound_outbound) and fd.sport >= 30000 and fd.sport <= 32767 and container and not nodeport_containers
   output: Unexpected K8s NodePort Connection (command=%proc.cmdline pid=%proc.pid connection=%fd.name container_id=%container.id image=%container.image.repository)
-  priority: NOTICE
+  priority: CRITICAL
   tags: [network, k8s, container, mitre_persistence, T1205.001]
 
 - list: network_tool_binaries
-  items: [nc, ncat, nmap, dig, tcpdump, tshark, ngrep, telnet, mitmproxy, socat, zmap]
+  items: [nc, ncat, nmap, tcpdump, tshark, ngrep, telnet, mitmproxy, socat, zmap]
```

Comment:
this is definitely a good point and we should reduce the false positive as much as possible.
Wondering if there might be a specific whitelist we can apply for dig used in OpenShift since this is the major use case.

Reply:
sure.. I will create separate PR for this... Closing this pull request .. For now only this change will work for me